### ISSUE TITLE
Does the screenshot tool use https://www.browserstack.com/docs/automate/selenium/take-screenshots#automatically-capture-screenshots?

### ISSUE THREAD

**Issue by white-than-wood** [scottgonzalez/node-browserstack#77]
Does the screenshot tool use https://www.browserstack.com/docs/automate/selenium/take-screenshots#automatically-capture-screenshots? There are only two channels of selenium and appium for screenshots of real devices, right?

**Comment by scottgonzalez** [comment]
No, this module does not control Selenium, it controls BrowserStack itself. `client.takeScreenshot` uses the [REST API](https://github.com/browserstack/api#screenshots) and the screenshot client uses the [Screenshot API](https://www.browserstack.com/screenshots/api).

**Comment by white-than-wood** [comment]
uh.......I mean what technology is the way Screenshot API takes screenshots based on.
